.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,51 @@
+<!--  Thanks for sending a pull request!  Here are some tips for you:
+1. Consider creating this PR as draft, for early feedback ping code owners
+2. Structure your PR accordingly to the focus areas, e.g. `api` containing all exposed API specs
+3. Write meaningful commit messages and PR description
+
+For more details, please check the [CONTRIBUTING](../CONTRIBUTING.md) guide.
+-->
+
+### What this PR does
+**Before this PR:**
+
+**After this PR:**
+
+Jira: <!-- optional: link to Jira issue -->
+Link to demo recording: <!-- optional: link to a demo recording -->
+
+### Breadcrumbs
+The following tradeoffs were made:
+
+The following alternatives were considered:
+
+Links to places where the discussion took place: <!-- optional: slack, GH issue, mailinglist, ... -->
```

Comment:
I think this discussion should be captured in JIRA
```suggestion
```

Reply:
Good point, updated.